### PR TITLE
Add recipe for pydash

### PR DESCRIPTION
**Pull request by souravsingh**

### CODE REVIEW

File: recipes/pydash/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  license_file: LICENSE.rst
+  summary: The kitchen sink of Python utility libraries for doing "stuff" in a functional way. 
+  description: "The kitchen sink of Python utility libraries for doing stuff in a functional way. Based on Lo-Dash JS library"
+  doc_url: https://pydash.readthedocs.io/en/latest/
+  dev_url: https://github.com/dgilland/pydash
+
+extra:
+  recipe-maintainers:
+    - souravsingh
+    - dgilland
```
Comment (synapticarbors) [inline recipes/pydash/meta.yaml]: @dgilland -- are you ok being added as a co-maintainer of the `pydash` package on conda-forge?

File: recipes/pydash/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
```
Comment (synapticarbors) [inline recipes/pydash/meta.yaml]: I think this package is a candidate for `noarch: python`

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

File: recipes/pydash/meta.yaml (hunk 1 above)
Comment (dgilland) [reply recipes/pydash/meta.yaml]: What does that involve? I'm not familiar with what responsibilities a maintainer/co-maintainer has with conda-forge.

Comment (dgilland) [reply recipes/pydash/meta.yaml]: That seems fine by me although my bandwidth is fairly low these days. But having @souravsingh as the primary would be best.

Comment (souravsingh) [reply recipes/pydash/meta.yaml]: I can take up the full responsibility of maintaining the feedstock.

**Pull request merged**